requirements-dev.txt

```diff
@@ -7,7 +7,9 @@ astropy>=0.0.dev0
 git+https://github.com/astropy/asdf-astropy
 git+https://github.com/spacetelescope/crds
 git+https://github.com/spacetelescope/gwcs
+numpy>=0.0.dev0
 git+https://github.com/astropy/photutils
+scipy>=0.0.dev0
```

Comment:
This change will not have any affect on the versions of `numpy` or `scipy` installed by `pip`. One needs to provide an alternate index for `pip` to search. See https://github.com/spacetelescope/romancal/pull/695#discussion_r1189078751 for details